crates/hc_sandbox/build.rs

```diff
@@ -0,0 +1,18 @@
+fn main() {
```

Comment:
I can figure out what this does, but a little comment would be welcome!

Reply:
```suggestion
/// There isn't a great way to get a path to the target directory at runtime,
/// so we build it in at compile time. The code generated by this build step
/// will only be used in integration tests, not the sandbox binaries themselves.
fn main() {
```